bundles/org.openhab.binding.shelly/README.md

```diff
@@ -80,35 +80,35 @@ See section [Discovery](#discovery) for details.
-| shellyplus1          | Shelly Plus 1 with 1x relay                              | SNSW-001X16EU, S3SW-001X16EU, S4SW-001X16EU |
-| shellyplus1pm        | Shelly Plus 1PM with 1x relay + power meter              | SNSW-001P16EU, S3SW-001P16EU, S4SW-001P16EU |
-| shellyplus2pm-relay  | Shelly Plus 2PM with 2x relay + power meter, relay mode  | SNSW-002P16EU, SNSW-102P16EU                |
-| shellyplus2pm-roller | Shelly Plus 2PM with 2x relay + power meter, roller mode | SNSW-002P16EU, SNSW-102P16EU                |
-| shellyplusplug       | Shelly Plug-S Gen 3, Shelly AZ Plug                      | SNPL-00112EU                                |
-| shellyplusplug       | Shelly Outdoor Plug-S                                    | S3PL-20112EU                                |
-| shellyplusplug       | Shelly Plug-IT                                           | SNPL-00110IT                                |
-| shellyplusplug       | Shelly Plug-UK                                           | SNPL-00112UK                                |
-| shellyplusplugus     | Shelly Plug-US                                           | SNPL-00116US                                |
-| shellyplusi4         | Shelly Plus i4 with 4x AC input                          | SNSN-0024X, S3SN-0024X                      |
-| shellyplusi4dc       | Shelly Plus i4 with 4x DC input                          | SNSN-0D24X                                  |
-| shellyplus10v        | Shelly Plus Dimmer 0/10V (Gen 2) or 0/1/10V (Gen 3)      | SNDM-00100WW, S3DM-0010WW                   |
-| shellyplusht         | Shelly Plus HT with temperature + humidity sensor        | SNSN-0013A                                  |
-| shellyhtg3           | Shelly Plus HT Gen 3 with temperature + humidity sensor  | S3SN-0U12A                                  |
-| shellyplussmoke      | Shelly Plus Smoke sensor                                 | SNSN-0031Z                                  |
-| shellypluswdus       | Shelly Plus Wall Dimmer US                               | SNDM-0013US                                 |
-| shellyplusrgbwpm     | Shelly Plus RGBW PM                                      | SNDC-0D4P10WW                               |
-| shellywalldisplay    | Shelly Plus Wall Display                                 | SAWD-0A1XX10EU1                             |
-| shellyblugw          | Shelly BLU Gateway                                       | SNGW-BT01                                   |
+| thing-type           | Model                                                    | Vendor ID                                                  |
+|----------------------|----------------------------------------------------------|------------------------------------------------------------|
+| shellyplus1          | Shelly Plus 1 with 1x relay                              | SNSW-001X16EU, S3SW-001X16EU, S3SW-001X16EU                |
```

Comment:
Why did you remove `S4SW-001X16EU` here? It's still mapped to `shellyplus1` in the code, and you even added missing coverage for this.

Reply:
fixed

Reply:
reverted